packages/graphback-runtime-mongodb/tests/MongoDataProviderTest.ts

```diff
@@ -213,4 +231,49 @@ describe('MongoDBDataProvider Basic CRUD', () => {
       expect(todos[t].text).toEqual(`todo${5 - t}`);
     }
   });
+
+  test('createdAt', async () => {
+    context = await createTestingContext(`
+    """
+    @model
+    @versioned
+    """
+    type Note {
+      id: ID!
+      text: String
+    }
+    `);
+    const cDate = new Date(2020, 5, 26, 18, 29, 23);
+    advanceTo(cDate);
+
+    const res = await context.providers.Note.create({ text: 'asdf' });
+    expect(res.createdAt).toEqual(cDate.getTime());
```

Comment:
We should test for updatedAt right? 

It is normal practice to have the `updatedAt` field set to `createdAt` field during creation.

Reply:
I think this functionality is tested in the test imediately after this

Reply:
Yes, we have the `updateAt test` but what I am asking is, should we also check that the `updateAt` is set immediately after creating an entity? I'd see that test here, and the `note.updatedAt` should be equal to `node.createdAt`.